doc/build-osx.md

```diff
@@ -5,7 +5,7 @@ This guide will show you how to build bitcoind (headless client) for OS X.
 Notes
 -----
 
-* Tested on OS X 10.7 through 10.10 on 64-bit Intel processors only.
+* Tested on OS X 10.7 through 10.11 on 64-bit Intel processors only.
```

Comment:
The build error in #7110 is unrelated to this change. Bitcoin builds fine on 10.11. The #7110 is very likely a QT issue. Will answer soon there.